agent/run_job.go

```diff
@@ -148,6 +159,50 @@ func (r *JobRunner) Run(ctx context.Context) error {
 	return nil
 }
 
+func (r *JobRunner) prependTimestampForLogs(s string, args ...any) []byte {
+	switch {
+	case r.conf.AgentConfiguration.ANSITimestamps:
+		return []byte(fmt.Sprintf(
+			"\x1b_bk;t=%d\x07%s",
+			time.Now().UnixNano()/int64(time.Millisecond),
+			fmt.Sprintf(s, args...),
+		))
+	case r.conf.AgentConfiguration.TimestampLines:
```

Comment:
Can we have a single stream for processing job output + stuff the job runner wants to pretend is job output? Aside from duplicating the timestamping logic, this could mess with the header times streamer, and might also skip the local log output file?

Reply:
I don't think we can have a single stream. The verification logs happen in the `agent start` process, while most of the other logs are streamed from the `agent bootstrap` process. So I think the best we can do is have two streams that are created from a common method.

We could have something in the local API so that the bootstrap process can stream logs over the local API to the remote agent API, but creating that for this task seems like a massive yak shave.

> pushing logs directly into r.logStreamer could make header times incorrect with non-ANSI timestamps
I don't really want to carry around the legacy of non-ANSI timestamps. Are you happy for me to remove support for these from this PR?

> might also skip the local log output file?
Yeah, they probably will. I'll look into something for this, and a way to not duplicate the timestamp logic, but if the answer to the question above is in the affirmative, then only about 1 line will be duplicated.

Reply:
> I don't think we can have a single stream. The verification logs happen in the agent start process, while most of the other logs are streamed from the agent bootstrap process. So I think the best we can do is have two streams that are created from a common method.

Looks like I was wrong about this. I'll do the work to make a single stream.